node/src/types/chainspec/highway_config.rs

```diff
@@ -59,22 +73,29 @@ impl ToBytes for HighwayConfig {
         let mut buffer = bytesrepr::allocate_buffer(self)?;
         buffer.extend(self.maximum_round_length.to_bytes()?);
         buffer.extend(self.reduced_reward_multiplier.to_bytes()?);
+        buffer.extend(self.performance_meter.blocks_to_consider.to_bytes()?);
```

Comment:
A nit:
Maybe this should be just 
```suggestion
        buffer.extend(self.performance_meter.to_bytes()?);
```
with a proper `bytesrepr` implementation for `PerformanceMeterConfig`?

Reply:
I'm not sure if this is worth it when it's just the one field - but I'd definitely support this way of doing it with more than one field.